desktop/cmp/contextmenu/StoreContextMenu.js

```diff
@@ -37,7 +37,10 @@ export class StoreContextMenu {
      *          `exportExcel` - same as above.
      *          `exportCsv` - export grid data to CSV via Hoist's server-side export capabilities.
      *          `exportLocal` - export grid data to Excel via ag-Grid's built-in client side export.
-     *          `'autosizeColumns` - autosize columns to fit their contents.
+     *          `autosizeColumns` - autosize columns to fit their contents.
+     *          `restoreDefaults` - restore grid defaults.
```

Comment:
Let's clarify a bit, briefly, as to what state is restored - column selection and ordering, grouping, and sort - something like that